From 21e9ef2001f8b10fef25e675b4be3bc9f1af8cfc Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Thu, 5 Jan 2012 11:42:35 -0800
Subject: [PATCH] compiler.h: Undef before redefining __attribute_const__

This is required to avoid warnings like
util/include/linux/compiler.h:8:0: error: "__attribute_const__" redefined [-Werror]

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 tools/include/linux/compiler.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/include/linux/compiler.h b/tools/include/linux/compiler.h
index 9c05a59f0184..4dbb2a0404bb 100644
--- a/tools/include/linux/compiler.h
+++ b/tools/include/linux/compiler.h
@@ -92,6 +92,7 @@
 #define __always_inline	inline
 #endif
 
+#undef __user
 #define __user
 #define __rcu
 #define __read_mostly
-- 
2.39.2

